CHANGELOG.md

```diff
@@ -381,6 +381,8 @@ Related changes include:
 - **decidim-core**: The `official_origin`, `participants_origin`, `user_group_origin` and `meeting_origin` scopes for the `Decidim::Authorable` and `Decidim::Coauthorable` concerns have been changed to `with_official_origin`, `with_participants_origin`, `with_user_group_origin` and `with_meeting_origin` respectively in [\#8748](https://github.com/decidim/decidim/pull/8748) for consistency. See the Searchlight removal change notes for reasoning.
 - **decidim-core**: Nicknames are now differents case insensitively, a rake task has been created to check every nickname and modify them if some are similar (Launch it with "bundle exec rake decidim:upgrade:fix_nickname_uniqueness"). Routing and mentions has been made case insensitive for every tab in profiles.
 
+### Changelog
```

Comment:
I understand the reasoning and I agree that we need an `h3` here, the thing is that the top `h1` is already "Change Log" (at line 1). How about something like "Detailed changes"? (Or whatever option you prefer)  

```suggestion
### Detailed changes
```

Alternatively, we could change the h1 but then it'd be weird to have a file called CHANGELOG and then the title doesn't correspond with that